### PR TITLE
Automatically import react when required

### PR DESCRIPTION
**Pull request by arush0311**
Fixes https://github.com/Pomax/react-formbuilder/issues/36

### CODE REVIEW

File: src/fields/CheckBoxGroup.jsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import { Component } from 'react';
+import React from 'react';
```
Comment (Pomax) [inline src/fields/CheckBoxGroup.jsx]: Why did you add this change?

Comment (arush0311) [inline src/fields/CheckBoxGroup.jsx]: https://facebook.github.io/react/docs/jsx-in-depth.html#react-must-be-in-scope

**Comment by arush0311** [comment]
updated

**Comment by Pomax** [comment]
Nice, testing now. However, a request for future PRs/branches: can you name them something that makes it obvious what they're for? For instance, `fix` is not a good branch name, it doesn't tell us anything, whereas `import-react` would be a great name, making it obvious in a branch list what it relates to.
